packages/backend/src/util/shared.ts

```diff
@@ -0,0 +1,16 @@
+export {
+  addClerkPrefix,
+  callWithRetry,
+  getClerkJsMajorVersionOrTag,
+  getScriptUrl,
+  isDevelopmentFromApiKey,
+  isProductionFromApiKey,
+  parsePublishableKey,
+} from '@clerk/shared';
+
+import { buildErrorThrower } from '@clerk/shared';
+// TODO: replace packageName with `${PACKAGE_NAME}@${PACKAGE_VERSION}` from tsup.config.ts
+export const errorThrower = buildErrorThrower({ packageName: '@clerk/backend' });
```

Comment:
I will fix this in another PR that will update how we build and run tests in backend package (currently tsup is not used in tests)